auto_editor/StructuredProjectSource.py

```diff
@@ -39,12 +39,20 @@ def get_all_dpct_file_paths(self) -> List[str]:
         :return: list with paths to all files inside directory at self.dpct_version_root
         '''
         # Todo: use same logic here as in PreAnalyser. Currently nested files aren't discovered.
-
+        dpct_extensions = ('*.dp.cpp', '*.dp.hpp')
+        all_dpct_files = []
         paths = []
-
-        for file_path in os.listdir(self.dpct_root):
-            if file_path.endswith(".dp.cpp") or file_path.endswith(".dp.hpp"):
-                paths.append(file_path)
+        test_path = str(self.dpct_root.stem)
```

Comment:
This is the path to the test project root now, but when we run CTA on other projects later on, it will be the project path. So maybe a more general name like `project_path` is better?